File: Game2/server/Game/Item/Food.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Game.Item
{
    public class Food : Item
    {
        private BFoodExtra extra;

        public Food(Game.Bag.BItem bItem, BFoodExtra extra) : base(bItem)
        {
            this.extra = extra;
        }

        public int Account => extra.Ammount;

        public override bool Use()
        {
            throw new NotImplementedException();
        }
    }
}

File: ZezexJava/server/src/Game/Item/Food.java
package Game.Item;

import Game.*;

public class Food extends Item {
	private BFoodExtra extra;

	public Food(Game.Bag.BItem bItem, BFoodExtra extra) {
		super(bItem);
		this.extra = extra;
	}

	public final int getAccount() {
		return extra.getAmmount();
	}

	@Override
	public boolean Use() {
		throw new UnsupportedOperationException();
	}
}